docs/markdown/reference/theming/README.md

```diff
@@ -44,7 +44,8 @@ $grid-breakpoints: (
   sm: 428px,
   md: 799px,
   lg: 899px,
-  xl: 1234px xxl: 1600px
+  xl: 1234px,
+  xxl: 1600px
```

Comment:
Actually, Boostrap-Vue doesn't support breakpoint names other than the standard bootstrap breakpoint names (xs, sm, md, lg, xl).

So it might be best to actually remove the `xxl` breakpoint from this example.

Reply:
@cmpscabral if you want to make that change, we can then merge this.